modules/ossm-about-smcp.adoc

```diff
@@ -21,3 +21,7 @@ Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where
 ====
 The {SMProductShortName} documentation uses `istio-system` as the example project, but you can deploy the service mesh to any project.
 ====
+
+ifndef::openshift-enterprise[]
+If you are deploying the control plane for use on Red Hat OpenShift on AWS (ROSA), see the https://access.redhat.com/solutions/6529231[KCS article] that discusses adding a new project and starting pods.
```

Comment:
```suggestion
If you are deploying the control plane for use on Red Hat OpenShift on AWS (ROSA), see the link:https://access.redhat.com/solutions/6529231[KCS article] that discusses adding a new project and starting pods.
```
- Should `Red Hat OpenShift on AWS (ROSA)` use an attribute?
- hyperlink should use `link:` at the beginning
- Per RHSSG, "Use the title of the Knowledgebase article for the link text, or use descriptive running text" : https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links

Reply:
Changes were made. 

I created a rosa attribute in common-attributes and added it to the service mesh section.

Reply:
@rh-tokeefe If this change is for ROSA only, why not use `ifdef::openshift-rosa`?